apps/challenges/utils.py

```diff
@@ -6,3 +8,9 @@
 get_challenge_model = get_model_object(Challenge)
 
 get_challenge_phase_model = get_model_object(ChallengePhase)
+
+
+def get_file_content(file_path, mode):
+    if os.path.isfile(file_path):
+        with open(file_path, mode) as file_content:
```

Comment:
The function intends to be general, thats great actually But here we are only reading the file and returning its content. If this is concerned, then I think mode should have a default value of read(`r`). Infact we can drop this mode as parameter if we are sure that there wont be any binary (mode=rb) file.

@deshraj : your thoughts here.